src/App.css

```diff
@@ -35,3 +34,57 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+/****** HOME PAGE STYLES ******/
+
+.raisedButton {
+    width: 200px;
+}
+.accountIcon {
+    margin-right: 2em;
+}
+
+.logo {
+    margin-left: 4em;
+    width: 4em;
+    padding: 5px;
+}
+
+.appBar {
+
+}
+
+.sidebar {
+    width: 12%;
+    height: 100vh;
+    border-right: 1px solid #dbdbdb;
+    float: left;
+}
+
+.mainContent {
+    display: inline-block;
+    margin: auto;
+    max-width: 88%;
+    height: 100%;
+    /*background-color: #dbdbdb;*/
```

Comment:
remove this if its going to be commented out.

Reply:
I'll be adding that color back in the next step, but can definitely get rid of it for now.